ai/scripts/test_rag_integration.py

```
@@ -0,0 +1,196 @@
+
+                if citations:
+                    print(f"  ✓ {len(citations)}건 검색됨")
+                    for i, cite in enumerate(citations[:2], 1):
+                        ref = cite.get("full_reference", "N/A")
+                        content = cite.get("content", "")[:80]
+                        print(f"    {i}. {ref}")
+                        print(f"       {content}...")
+                else:
+                    print("  ✗ 검색 결과 없음")
+
+            except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)